.github/workflows/ci-tests_blog.yml

```diff
@@ -0,0 +1,26 @@
+name: Unit tests
```

Comment:
I believe the naming of the file is unfortunate. Either you should rename the file that test the helloworld as well. Either you should put everything under `ci-tests.yml`.